packages/next/build/utils.ts

```diff
@@ -1259,7 +1260,13 @@ export async function isPageStatic({
         const runtime = await getRuntimeContext({
           paths: edgeInfo.files.map((file: string) => path.join(distDir, file)),
           env: edgeInfo.env,
-          edgeFunctionEntry: edgeInfo,
+          edgeFunctionEntry: {
+            ...edgeInfo,
+            wasm: (edgeInfo.wasm ?? []).map((binding: AssetBinding) => ({
+              ...binding,
+              filePath: path.join(distDir, binding.filePath),
+            })),
+          },
```

Comment:
good catch <picture data-single-emoji=":clapping-inclusive:" title=":clapping-inclusive:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..INxJUaQ7EPjXT8iU.83ogDs_9mZMQCFU5D1RGTWOFIoN63fzdOBnLxc-NGjTXQfC3NfwQnAstW0Z8C9lI-Fo9e7sWINan_j_rOx88K-ZfJNYhALimV_gjS6e40mw5u_zt6-spKwo0A_iP5MeaKgMD0g.woLtdwyQJwzk0UN3lTRmRA" alt=":clapping-inclusive:" align="absmiddle"></picture>

Reply:
I wonder if we need to do the same for blob assets? (i.e. `fetch(new URL("./file.png", import.meta.url))`)

anyway that can be in a new PR <picture data-single-emoji=":open_source_parrot:" title=":open_source_parrot:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..Q1IeyQlQf8x5YoIY.H4HOl3zZaZSImYyyi7wNmOh2nkP5V4HKQjbreFXlr4kpjYBtkVBliS5vr1XPrq4M6xEF6t7sEjmyCnqZYduLprbIKVywNTiXUEhS26esgolFD5DE8n6UMgtPZO72U57ipTTfbw.rh7S8LLisFckLY1UJpsyhQ" alt=":open_source_parrot:" align="absmiddle"></picture>

Reply:
Good question 🤔 yeah I can take a look and add a test case as a follow-up!